src/test/java/com/returners/movies/controller/MovieControllerTests.java

```diff
@@ -73,6 +74,36 @@ public void testGetAllMoviesReturnsMovies() throws Exception {
             .andExpect(MockMvcResultMatchers.jsonPath("$.data[2].title").value("The Bucket List"));
   }
 
+    @Test
+    public void testGetMoviesBySearch() throws Exception {
+
+        List<Movie> movies = new ArrayList<>();
+        String[] actors = {"Keira Knightley", "Ralph Fiennes", "Dominic Cooper"};
+        Movie movie = new Movie(1L, actors, 6, "The Duchess", 2008, new Certification(4L,"12A"), new Genre(9L,"Drama"));
+        movies.add(movie);
+        actors = new String[]{"Tim Robbins","Morgan Freeman","Bob Gunton"};
+        movie= new Movie(2L, actors, 9, "The Shawshank Redemption", 1994, new Certification(8L,"R"), new Genre(3L,"Comedy"));
+        movies.add(movie);
+        actors = new String[]{"Jack Nicholson", "Morgan Freeman", "Sean Hayes"};
+        movie = new Movie(3L, actors, 7, "The Bucket List", 2007, new Certification(4L,"12A"), new Genre(4L,"Horror"));
+        movies.add(movie);
+
```

Comment:
You can create a separate private method to return List<Movies> and use that method in different test cases instead of repeating this code.